### PR TITLE
Adding a new extension Slicer Cervical Spine

### PR DESCRIPTION
**Pull request by idhamari**
# Submitting a new extension

Thank you very much for considering sharing your extension with the Slicer community.

To make sure users can find your extension, understand what it is intended for and how to use it, please complete the checklist below. You do not need to complete all the item by the time you submit the pull request, but most likely the changes will only be merged if all the tasks are done.

- [x] Extension has a reasonable name (not too general, not too narrow, suggests what the extension is for)
- [x] Repository name is Slicer+ExtensionName
- [x] Extension description summarizes in 1-2 sentences what the extension is usable (should be understandable for non-experts)
- [x] Extension URL and revision (scmurl, scmrevision) is correct, consider using a branch name (master, release, ...) instead of a specific git has to avoid re-submitting pull request whenever the extension is updated
- [x] Extension icon URL is correct
- [x] Screenshot URLs (screenshoturls) are correct, contains at least one
- [x] Homepage URL points to valid webpage containing the following:
- [x] Extension name
- [x] Short description: 1-2 sentences, which summarizes what the extension is usable for
- [x] At least one nice, informative image, that illustrates what the extension can do. It may be a screenshot.
- [x] Description of contained modules: at one sentence for each module
- [x] Tutorial: step-by-step description of at least the most typical use case, include a few screenshots, provide download links to sample input data set
- [x] Publication: link to publication and/or to PubMed reference (if available)

See more information about the submission process on the Slicer wiki: https://www.slicer.org/wiki/Documentation/Nightly/Developers/Tutorials/BuildTestPackageDistributeExtensions

If you have any questions or comments then please describe them here.

### CODE REVIEW

**Comment by pieper** [comment]
LGTM - thanks!

**Comment by pieper** [comment]
One thing @idhamari , [recently we've been discussing extension licenses](https://github.com/Slicer/ExtensionsIndex/pull/1616).  Could you add a License.txt file to your repository for clarity?

**Comment by idhamari** [comment]
Thanks for the info. A license file is added and Slicer license is mentioned in the readme file.

File: SlicerCervicalSpine.s4ext [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# list dependencies
+# - These should be names of other modules that have .s4ext files
+# - The dependencies will be built first
+depends NA
+
+# Inner build directory (default is ".")
+build_subdirectory .
+
+# homepage
+homepage  https://github.com/MedicalImageAnalysisTutorials/SlicerCervicalSpine/blob/master/docs/index.html
```
Comment (fedorov) [inline SlicerCervicalSpine.s4ext]: This does not seem to be the correct URL. Do you have a web site hosted by GitHub with the rendered version of this item?

**Comment by idhamari** [comment]
sorry for that, I added a rendered webpage.

**Comment by idhamari** [comment]
It's been 10 days, is there still something missing?

**Comment by jcfr** [comment]
It was just integrated. In the mean time, could you update the icon from 32x32 to 128x128.

It is currently quite small:

iconurl https://raw.githubusercontent.com/MedicalImageAnalysisTutorials/SlicerCervicalSpine/master/SlicerCervicalSpine.png

**Comment by idhamari** [comment]
Thanks. Icon size is changed to 128x128.

**Pull request merged**